### PR TITLE
NOISSUE - Exit BS procedure upon successful bootstrap retrieval

### PR DESCRIPTION
**Pull request by mteodor**
Minor fix to exit upon succesfull bootstrap and change struct tags

### CODE REVIEW

File: internal/pkg/bootstrap/bootstrap.go [hunk 1]
```diff
@@ -58,16 +58,18 @@ func Bootstrap(cfg Config, logger log.Logger, file string) error {
 
 	dc := deviceConfig{}
 	for i := 0; i < int(retries); i++ {
+
```
Comment (drasko) [inline internal/pkg/bootstrap/bootstrap.go]: Kill this white line

File: internal/pkg/bootstrap/bootstrap.go [hunk 2]
```diff
@@ -58,16 +58,18 @@ func Bootstrap(cfg Config, logger log.Logger, file string) error {
...
-				logger.Info(fmt.Sprintf("Continuing with local config"))
-				return nil
-			}
+		if err == nil {
+			break
+		}
+		logger.Error(fmt.Sprintf("Fetching bootstrap failed with error: %s", err))
+		logger.Debug(fmt.Sprintf("Retries remaining: %d. Retrying in %d seconds", retries, retryDelaySec))
+		time.Sleep(time.Duration(retryDelaySec) * time.Second)
+		if i == int(retries)-1 {
```
Comment (drasko) [inline internal/pkg/bootstrap/bootstrap.go]: Maybe it makes sense to consider exponential backoff: https://github.com/cenkalti/backoff

Comment (mteodor) [reply internal/pkg/bootstrap/bootstrap.go]: can we make an issue for this and implement this later?

Comment (drasko) [reply internal/pkg/bootstrap/bootstrap.go]: Yes sure.

**Pull request merged**